Resources/Prototypes/Entities/Clothing/Hands/colored.yml

```diff
@@ -191,3 +191,11 @@
       - 1.5
       - 1.5
       - 1.5
+
+- type: entity
+  parent: ClothingHandsGlovesColorYellow
+  id: ClothingHandsGlovesConducting
+  suffix: Conducting
+  components:
+  - type: Insulated
+    coefficient: 5 # zap em good
```

Comment:
You are going to hell

Reply:
Sidenote, they will show up on forensics scanners as Insulative Fibers.

Reply:
yes its a stealth item